packages/dso-toolkit/components/Componenten/_button/button.njk

```diff
@@ -2,7 +2,9 @@
   {{- attributes(
     [roleDescription, 'aria-roledescription', roleDescription],
     [disabled, 'disabled'],
-    [ariaDescribedby, 'aria-describedby', ariaDescribedby]
+    [ariaDescribedby, 'aria-describedby', ariaDescribedby],
+    [ariaPopup, 'aria-haspopup', ariaPopup],
```

Comment:
De attribuut is `aria-haspopup`, variabele niet afkorten maar ook zo noemen: `ariaHasPopup`

Reply:
aangepast